doc/src/driver/compile.rst

```diff
@@ -28,6 +28,10 @@ Enable implicit offload copying
 
 Disable fast math optimization
 
+.. option:: --exhaustive-tune
+
+Perform an exhaustive search to find the fastest version of generated kernels for selected backend
```

Comment:
We could also use this flag to time different graph transformations as well, not just kernels.

Reply:
`-tune` still sounds better to me. If we are timing different graph transformations/optimization then `tune` relates to auto-tuning.  IMO, also during the discussions, tuning has lesser overloaded meaning compared to optimize.